taskcluster/tc-asserts.sh

```diff
@@ -533,6 +533,15 @@ run_hotword_tests()
   assert_correct_ldc93s1_lm "${hotwords_decode}" "$status"
 }
 
+run_android_hotword_tests()
+{
+  set +e
+  hotwords_decode=$(${DS_BINARY_PREFIX}deepspeech --model ${DATA_TMP_DIR}/${model_name} --scorer ${DATA_TMP_DIR}/kenlm.scorer --audio ${DATA_TMP_DIR}/${ldc93s1_sample_filename} --hot_words "foo:0.0,bar:-0.1" 2>${TASKCLUSTER_TMP_DIR}/stderr)
```

Comment:
pretty sure you want `${model_name_mmap}` here

Reply:
added